IntegrationTests/Assets/APITests/PurchasesAPITests.cs

```diff
@@ -16,20 +16,20 @@ public class PurchasesAPITests : MonoBehaviour
     private void Start()
     {
         Purchases purchases = GetComponent<Purchases>();
-        purchases.RevenueCatAPIKeyApple = "def";
-        purchases.RevenueCatAPIKeyGoogle = "ghi";
-        purchases.RevenueCatAPIKeyAmazon = "ghi";
-        purchases.UseAmazon = true;
+        purchases.revenueCatAPIKeyApple = "def";
+        purchases.revenueCatAPIKeyGoogle = "ghi";
+        purchases.revenueCatAPIKeyAmazon = "ghi";
+        purchases.useAmazon = true;
```

Comment:
public properties that are visible to the Unity Editor use camelCase for whatever reason